webapp/packages/plugin-data-viewer-result-trace-details/src/DVResultTraceDetailsPresentation.tsx

```diff
@@ -52,22 +26,41 @@ export const DVResultTraceDetailsPresentation: DataPresentationComponent = obser
   const state = useResultTraceDetails(model, resultIndex);
 
   useAutoLoad(DVResultTraceDetailsPresentation, state, undefined, undefined, true);
+  const trace = state.trace;
 
-  if (!state.trace?.length) {
+  if (!trace?.length) {
     return <TextPlaceholder>{translate('plugin_data_viewer_result_trace_no_data_placeholder')}</TextPlaceholder>;
   }
 
+  function getCell(rowIdx: number, colIdx: number) {
+    switch (colIdx) {
+      case 0:
+        return trace![rowIdx]?.name ?? '';
+      case 1:
+        return trace![rowIdx]?.value ?? '';
+      case 2:
+        return trace![rowIdx]?.description ?? '';
+    }
+
+    return '';
+  }
```

Comment:
very unusual way of setting the columns. Can we rearrange the columns order in grid? Will it work?